### ISSUE TITLE
Sandbox: Check $TMP permissions

### ISSUE THREAD

**Issue by edwardcallahan** [typesafehub/conductr-cli#290]
If the sandbox command is unable to create the ip named folder in $TMP, it fails to start w/out any indication of the issue:
```
Starting ConductR agent instance 2 on 192.168.10.3..
Waiting for ConductR to start..............................
|------------------------------------------------|
| Summary                                        |
|------------------------------------------------|
Error: ConductR has not been started within 60.0 seconds.
Error: Set the env CONDUCTR_SANDBOX_WAIT_RETRY_INTERVAL to increase the wait timeout.
```

Running the sandbox image out of .conductr, `/.conductr $ images/core/bin/conductr -Dconductr.ip=192.168.10.1` reveals the permissions issue.

```
2017-02-01T18:55:38Z AJGoddard ERROR ActorSystemImpl [sourceThread=main, akkaTimestamp=18:55:38.522UTC, akkaSource=akka.actor.ActorSystemImpl(conductr), sourceActorSystem=conductr] - Failure to create storage directory: /tmp/conductr/192.168.10.1/bundles - unable to proceed
java.nio.file.AccessDeniedException: /tmp/conductr/192.168.10.1
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.checkAccess(UnixFileSystemProvider.java:308)
	at java.nio.file.Files.createDirectories(Files.java:746)
	at com.typesafe.conductr.ConductR$$anonfun$ensureExistsAndWriteable$1$2.apply(ConductR.scala:275)
	at com.typesafe.conductr.ConductR$$anonfun$ensureExistsAndWriteable$1$2.apply(ConductR.scala:275)
	at scala.util.Try$.apply(Try.scala:192)
	at com.typesafe.conductr.ConductR$.ensureExistsAndWriteable$1(ConductR.scala:275)
	at com.typesafe.conductr.ConductR$.createDirectories(ConductR.scala:286)
	at com.typesafe.conductr.ConductR$.startup(ConductR.scala:172)
	at com.typesafe.conductr.ConductR$.com$typesafe$conductr$ConductR$$run(ConductR.scala:112)
	at com.typesafe.conductr.ConductR$$anonfun$main$1.apply(ConductR.scala:109)
	at com.typesafe.conductr.ConductR$$anonfun$main$1.apply(ConductR.scala:109)
	at scala.Option.foreach(Option.scala:257)
	at com.typesafe.conductr.ConductR$.main(ConductR.scala:109)
	at com.typesafe.conductr.ConductR.main(ConductR.scala)
```

**Comment by huntc** [comment]
Yes, it would be great to check the permissions and halt if they are wrong.

**Comment by huntc** [comment]
...or use `tmpdir`

**Comment by huntc** [comment]
Guide the user to use the `sandbox logs` command.

**Comment by longshorej** [comment]
Fixed via #295
